paddle/fluid/operators/prroi_pool_op.cu

```diff
@@ -281,17 +306,40 @@ class GPUPRROIPoolGradOpKernel : public framework::OpKernel<T> {
       rois_batch_id_list.Resize({rois_num});
       int* rois_batch_id_data =
           rois_batch_id_list.mutable_data<int>(platform::CPUPlace());
-      auto rois_lod = rois->lod().back();
-      int rois_batch_size = rois_lod.size() - 1;
-      for (int n = 0; n < rois_batch_size; ++n) {
-        for (size_t i = rois_lod[n]; i < rois_lod[n + 1]; ++i) {
-          rois_batch_id_data[i] = n;
+
+      if (ctx.HasInput("BatchRoINums") || rois->lod().empty()) {
```

Comment:
Errors should be reported if  `rois->lod().empty() == true` but `ctx.HasInput("BatchRoINums") == false`

Reply:
Done! thanks